packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -59,6 +60,21 @@ export const flushFilePath = (filePath: string): string => {
   return nativePath;
 };
 
+const isSymbolicLink = (path: string) => {
+  try {
+    let currentPath = resolve(path);
+    while(currentPath !== '/') {
```

Comment:
Not sure the while condition will work on windows.

Reply:
Thanks I just updated. The root path of windows is not similar with mac.